android/src/main/java/vn/luongvo/kmm/survey/android/ui/screens/home/HomeViewModel.kt

```diff
@@ -34,6 +38,7 @@ class HomeViewModel(
             _currentDate.emit(
                 dateFormatter.format(Calendar.getInstance().time, HeaderDateFormat)
             )
+            _appVersion.emit("v${BuildConfig.VERSION_NAME} (${BuildConfig.VERSION_CODE})")
```

Comment:
For the first time open the app, we will show the login screen but this one is fired, right?

Reply:
Fixed in https://github.com/luongvo/kmm-survey/pull/72/commits/f70e87b2b4395755931ced93ce791fa288907f10